DllImportGenerator/DllImportGenerator.UnitTests/TestUtils.cs

```diff
@@ -16,6 +18,25 @@ namespace DllImportGenerator.UnitTests
 {
     internal static class TestUtils
     {
+        private static string NuGetConfigPath { get; } = FindRepoNuGetConfig();
+
+        private static string FindRepoNuGetConfig()
+        {
+            ReadOnlySpan<char> assemblyLocation = typeof(TestUtils).Assembly.Location;
+            for (ReadOnlySpan<char> directory = Path.GetDirectoryName(assemblyLocation); !directory.IsEmpty; directory = Path.GetDirectoryName(directory))
+            {
+                string nugetConfigPath = Path.Join(directory, "NuGet.config");
+                if (File.Exists(nugetConfigPath))
+                {
+                    return nugetConfigPath;
+                }
+            }
+
+            Debug.Assert(false, "This repo should always contain a NuGet.config at the repo root.");
```

Comment:
Could we do something like make the `DllImportGenerator.UnitTests` build such that it always copies the NuGet.config at the repo root to its output? So that we avoid the search and could allow copying/running the test outside the repo.

Reply:
Yeah I think we could do that instead. I'll try it out.